cdap-ui/app/features/tracker/controllers/results-ctrl.js

```diff
@@ -150,6 +150,9 @@ class TrackerResultsController {
         createDate: entity.metadata.SYSTEM.properties['creation-time'],
         datasetType: entity.metadata.SYSTEM.properties.type
       });
+      if(entity.metadata.SYSTEM.tags.indexOf('explore') !== -1) {
```

Comment:
I think we should use the same logic here, that we use to determine whether we want to show the 'explore' tab here (screenshot below). @ajainarayanan thoughts?

![image](https://cloud.githubusercontent.com/assets/2440977/16473192/de3295fa-3e1f-11e6-9bf9-6507f3f41d8c.png)